pkg/networkservice/mechanisms/kernel/kerneltap/client.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2020-2021 Cisco and/or its affiliates.
```

Comment:
The code of mechanism chain element becomes too complex. Can we move all dump logic into separate chain element `dumpkerneltap`? We can put the element into  `pkg/networkservice/mechanisms/kernel/kerneltap/dumpkerneltap` and use together with kerneltap (See at exa ple below)